src/Nethermind/Nethermind.Taiko/InitializeBlockchainTaiko.cs

```diff
@@ -54,18 +55,17 @@ protected override BlockProcessor CreateBlockProcessor(BlockCachePreWarmer? preW
         if (_api.BlockTree is null) throw new StepDependencyException(nameof(_api.BlockTree));
         if (_api.EthereumEcdsa is null) throw new StepDependencyException(nameof(_api.EthereumEcdsa));
 
-        return new BlockProcessor(
-            _api.SpecProvider,
+        return new BlockProcessor(_api.SpecProvider,
             _api.BlockValidator,
             _api.RewardCalculatorSource.Get(transactionProcessor),
             new BlockInvalidTxExecutor(new ExecuteTransactionProcessorAdapter(transactionProcessor), worldState),
             worldState,
-            _api.ReceiptStorage,
-            transactionProcessor,
+            _api.ReceiptStorage!,
             new BeaconBlockRootHandler(transactionProcessor, worldState),
             new BlockhashStore(_api.SpecProvider, worldState),
             _api.LogManager,
-            new BlockProductionWithdrawalProcessor(new NullWithdrawalProcessor()),
+            new BlockProductionWithdrawalProcessor(new NullWithdrawalProcessor()), // Dude, why is this block **production** withdrawal processor.
```

Comment:
@flcl42 ?

Reply:
We do not use block production in taiko. We build lists in a separate method, see `taikoAuth_txPoolContentWithMinTip`

Reply:
Seems like can be just WithdrawalProcessor